week-2/14-align-self/readme.md

```diff
@@ -0,0 +1,13 @@
+# Align self
+
+With `align-items` you move all the items at once (on the **non-main axis**).
+With `align-self` you move only one of the items.
+
+![Example of align-items vs. align-self](./ignore_this_folder/example1.png)
```

Comment:
I drew this up to try and illustrate them against the axes.

![flexbox-axis-align](https://user-images.githubusercontent.com/2306629/50686882-f4915180-1016-11e9-82d2-b0a374294676.png)

I've kept the source file for it, so if you want me to draw up any more for describing the main axis/cross axis, just sketch out something quick and I can reproduce it. :+1:

Reply:
Much better than my example 😃